lib/livebook/notebook/explore/intro_to_kino.livemd

```diff
@@ -0,0 +1,162 @@
+# Interactions with Kino
+
+## Setup
+
+In this notebook we will explore the possibilities that
+[`kino`](https://github.com/elixir-nx/kino) brings
+into your notebooks. Kino can be thought of as Livebook's
+friend that instructs it how to render certain output
+and interact with it.
+
+```elixir
+Mix.install([
+  {:kino, "~> 0.1.0"},
+  {:vega_lite, "~> 0.1.0"}
+])
+```
+
+```elixir
+alias VegaLite, as: Vl
+```
+
+## VegaLite
+
+In the [Plotting with VegaLite](/explore/notebooks/intro-to-vega-lite) notebook we show
+numerous ways in which you can visualize your data. However, all of the plots
+there are static.
+
+Using Kino, we can dynamically stream data to the plot, so that it keeps updating!
+To do that, all you need is a regular VegaLite specification that you then pass
+to `Kino.VegaLite.start/1`. You don't have to specify any data up-front.
+
+```elixir
+widget =
+  Vl.new(width: 400, height: 400)
+  |> Vl.mark(:line)
+  |> Vl.encode_field(:x, "x", type: :quantitative)
+  |> Vl.encode_field(:y, "y", type: :quantitative)
+  |> Kino.VegaLite.start()
+```
+
+Then you can push data to the plot widget at any point and see it update dynamically:
+
+```elixir
+for i <- 1..300 do
+  point = %{x: i / 10, y: :math.sin(i / 10)}
+
+  # The :window option ensures we only show the latest
+  # 100 data points on the plot
+  Kino.VegaLite.push(widget, point, window: 100)
+
+  Process.sleep(25)
+end
+```
+
+You can also explicitly clear the data:
+
+```elixir
+Kino.VegaLite.clear(widget)
+```
+
+## Kino.render/1
```

Comment:
Maybe move this section to the end and say that we can use it to print _any_ of the widgets in this chapter as well as custom data structures?